docker/php/php.ini

```diff
@@ -0,0 +1,8 @@
+; Set custom PHP configuration
+; Default values: https://github.com/php/php-src/blob/master/php.ini-development
+
+; Increase memory limit for composer
+memory_limit = 2G
```

Comment:
I missing some optimizations here to avoid performance issues and I would increase the upload size:

```ini
upload_max_filesize = 510M
post_max_size = 512M

# https://symfony.com/doc/current/performance.html#configure-the-php-realpath-cache
realpath_cache_size=4096K
realpath_cache_ttl=600

# https://symfony.com/doc/current/performance.html#configure-opcache-for-maximum-performance
[opcache]
opcache.enable=1
; maximum memory that OPcache can use to store compiled PHP files
opcache.memory_consumption=256
; maximum number of files that can be stored in the cache
opcache.max_accelerated_files=20000
```

Reply:
would you do this in a development environment too?

Reply:
also i am a bit afraid of enabling opcache because this might lead to confusing errors for beginners, doesnt it? 🙂

Reply:
we could add `opcache.validate_timestamps = 1` and `opcache.revalidate_freq = 2` which are the default values. to be sure that the opcache is valdiated every 2 seconds.